packages/frontend-shared/src/components/Button.vue

```diff
@@ -88,6 +88,7 @@ const VariantClassesTable = {
   linkBold: 'border-transparent text-indigo-500 font-medium',
   text: 'border-0',
   secondary: 'bg-jade-500 text-white hocus-secondary',
+  white: 'bg-white text-indigo-500 font-medium hocus-default',
```

Comment:
Update the CT test for the Button to add a case for this new variant.  There is a `playground` test in the spec that showcases all the variants.